tests/test_init.py

```diff
@@ -19,90 +19,241 @@
+                       (2, 1),),
+                      (qml.init.cvqnn_layers_r_normal,
+                       {'n_layers': 2, 'n_wires': 3, 'mean': 0, 'std': 1},
+                       (2, 3),),
+                      (qml.init.cvqnn_layers_r_normal,
+                       {'n_layers': 2, 'n_wires': 1, 'mean': 0, 'std': 1},
+                       (2, 1),),
+                      (qml.init.cvqnn_layers_phi_r_normal,
+                       {'n_layers': 2, 'n_wires': 3, 'mean': 0, 'std': 1},
+                       (2, 3),),
+                      (qml.init.cvqnn_layers_phi_r_normal,
+                       {'n_layers': 2, 'n_wires': 1, 'mean': 0, 'std': 1},
+                       (2, 1),),
+                      (qml.init.cvqnn_layers_a_normal,
+                       {'n_layers': 2, 'n_wires': 3, 'mean': 0, 'std': 1},
+                       (2, 3),),
+                      (qml.init.cvqnn_layers_a_normal,
+                       {'n_layers': 2, 'n_wires': 1, 'mean': 0, 'std': 1},
+                       (2, 1),),
+                      (qml.init.cvqnn_layers_phi_a_normal,
+                       {'n_layers': 2, 'n_wires': 3, 'mean': 0, 'std': 1},
+                       (2, 3),),
+                      (qml.init.cvqnn_layers_phi_a_normal,
+                       {'n_layers': 2, 'n_wires': 1, 'mean': 0, 'std': 1},
+                       (2, 1),),
+                      (qml.init.cvqnn_layers_kappa_normal,
+                       {'n_layers': 2, 'n_wires': 3, 'mean': 0, 'std': 1},
+                       (2, 3),),
+                      (qml.init.cvqnn_layers_kappa_normal,
+                       {'n_layers': 2, 'n_wires': 1, 'mean': 0, 'std': 1},
+                       (2, 1),),
+                      (qml.init.interferometer_theta_normal,
+                       {'n_wires': 3, 'mean': 0, 'std': 1},
+                       (3,)),
+                      (qml.init.interferometer_theta_normal,
+                       {'n_wires': 1, 'mean': 0, 'std': 1},
+                       (0,)),
+                      (qml.init.interferometer_phi_normal,
+                       {'n_wires': 3, 'mean': 0, 'std': 1},
+                       (3,)),
+                      (qml.init.interferometer_phi_normal,
+                       {'n_wires': 1, 'mean': 0, 'std': 1},
+                       (0,)),
+                      (qml.init.interferometer_varphi_normal,
+                       {'n_wires': 3, 'mean': 0, 'std': 1},
+                       (3,)),
+                      (qml.init.interferometer_varphi_normal,
+                       {'n_wires': 1, 'mean': 0, 'std': 1},
+                       (1,)),
+                      (qml.init.random_layers_uniform,
+                       {'n_layers': 2, 'n_wires': 3, 'n_rots': 10, 'low': 0, 'high': 1},
+                       (2, 10)),
+                      (qml.init.random_layers_uniform,
+                       {'n_layers': 2, 'n_wires': 3, 'n_rots': None, 'low': 0, 'high': 1},
+                       (2, 3)),
```

Comment:
Just to have a case of falling back to `n_wires=1` due to `n_rots is None`
```suggestion
                       (2, 3)),
                       (qml.init.random_layers_uniform,
                       {'n_layers': 2, 'n_wires': 1, 'n_rots': None, 'low': 0, 'high': 1},
                       (2, 1)),
```